pkg/controllers/report/resource/controller.go

```diff
@@ -320,7 +320,8 @@ func (c *controller) updateDynamicWatchers(ctx context.Context) error {
 		for _, policy := range vapPolicies {
 			kinds, err := admissionpolicy.GetKinds(policy.Spec.MatchConstraints, restMapper)
```

Comment:
The `GetKinds` method short-circuits once the first `resolveKind` fails.
This worked if the error is considered fatal.
However now that we just want to log it as an error this could be an issue:
A policy could reference multiple kinds and if the first one fails to be resolved any following valid resources would be silently ignored. Also any errors in the following resources would not be logged.